src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2390,6 +2403,444 @@ def remove_triggered_webjob(cmd, resource_group_name, name, webjob_name, slot=No
     return client.web_apps.delete_triggered_web_job(resource_group_name, name, webjob_name)
 
 
+def webapp_list_hc(cmd, name, resource_group, slot=None):
```

Comment:
this and functionapp method for the same are almost identical - is there any reason we can have both combined to a single method?